stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import Components from "stockflux-components";
+import "./SecuritiesTable.css";
+
+// hard coded data to be replaced with data from backend
+let SECURITIES_DATA = [
```

Comment:
If this variable is not intended to be reassigned then it is a good idea to use `const` instead of `let`

Reply:
okay, done